wandb/sdk/wandb_settings.py

```diff
@@ -1738,14 +1737,6 @@ def _infer_settings_from_environment(
         if self.host is None:
             settings["host"] = socket.gethostname()  # type: ignore
 
-        if self.username is None:
-            try:  # type: ignore
-                settings["username"] = getpass.getuser()
-            except KeyError:
-                # getuser() could raise KeyError in restricted environments like
-                # chroot jails or docker containers. Return user id in these cases.
-                settings["username"] = str(os.getuid())
```

Comment:
why remove all this logic?

Reply:
because it is wrong... username is used both as user on a system and name of the user in wandb